component_generator.thor

```diff
@@ -83,4 +84,8 @@ class ComponentGenerator < Thor::Group
   def js_package_name
     options[:js]
   end
+
+  def inline?
+    options[:inline]
```

Comment:
Might we want to document this option somewhere?

Reply:
added a description!